### ISSUE TITLE
The dimension of the output of the discriminator

### ISSUE THREAD

**Issue by ghost** [LynnHo/CycleGAN-Tensorflow-Simple#2]
Hello!

I find the dimension of the output of the discriminator is # h4 is (32 x 32 x 1), and then the code calculate the loss ：

    a2b_dis = models.discriminator(a2b, 'b', reuse=True)
    # losses
    g_loss_a2b = tf.identity(ops.l2_loss(a2b_dis, tf.ones_like(a2b_dis)), name='g_loss_a2b')

I am so confused,  as I think the the dimension of the output of the discriminator should be 1.

Could you please give some hints? 

THX

**Comment by LynnHo** [comment]
@Simon4john The author's pytorch code uses this strategy. https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix. I don't know exactly why, but I think it's used for more stable training comparing to one dimension output.
